holoviews/core/util.py

```diff
@@ -647,6 +647,30 @@ def walk_depth_first(name):
                                     (names_by_level.get(i, None)
                                      for i in itertools.count())))
 
+
+def is_cyclic(graph):
+    """Return True if the directed graph g has a cycle."""
```

Comment:
Right, all three methods here (``sort_topologically``, ``cyclical`` and ``one_to_one``) use the same representation, which is mapping between nodes and edges, will add the docstring.